### ISSUE TITLE
[Bug] Lag spikes 

### ISSUE THREAD

**Issue by TisRyno** [Azim/LethalPresents#5]
Using the latest version 1.0.6

When playing any map, the mod has the below error at regular intervals causing mini-lag spikes.

```
[Info   :LethalPresents] List of spawnable enemies (outside):
[Error  : Unity Log] NullReferenceException
Stack trace:
UnityEngine.Object.get_name () (at <e27997765c1848b09d8073e5d642717a>:IL_0001)
LethalPresents.LethalPresentsPlugin+<>c.<updateCurrentLevelInfo>b__14_1 (SpawnableEnemyWithRarity e) (at <d2e304e6603d4c5bac7acd0485825dba>:IL_0000)
System.Collections.Generic.List`1[T].ForEach (System.Action`1[T] action) (at <787acc3c9a4c471ba7d971300105af24>:IL_001E)
LethalPresents.LethalPresentsPlugin.updateCurrentLevelInfo (On.RoundManager+orig_AdvanceHourAndSpawnNewBatchOfEnemies orig, RoundManager self) (at <d2e304e6603d4c5bac7acd0485825dba>:IL_00AC)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Hook<RoundManager::AdvanceHourAndSpawnNewBatchOfEnemies>?-715345556(RoundManager)
(wrapper dynamic-method) RoundManager.DMD<RoundManager::Update>(RoundManager)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Trampoline<RoundManager::Update>?582995046(RoundManager)
LethalThings.Patches.Debug.RoundManager_Update (On.RoundManager+orig_Update orig, RoundManager self) (at O:/github/LethalThings/LethalThings/Patches/Debug.cs:121)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Hook<RoundManager::Update>?555988936(RoundManager)
```

**Comment by Azim** [comment]
Please provide the full log file.
It seems some mod is overriding my hooks, which causes what was supposed to be ran only once to trigger every time game spawns enemies instead

**Comment by Azim** [comment]
should be fixed in 1.0.7
